include/libkrun.h

```diff
@@ -283,6 +364,8 @@ int32_t krun_add_virtiofs2(uint32_t ctx_id,
 int32_t krun_set_passt_fd(uint32_t ctx_id, int fd);
 
 /**
+ * DEPRECATED. Use krun_add_net_gvproxy instead.
```

Comment:
You forgot to rename the reference to function in the doc:
```suggestion
 * DEPRECATED. Use krun_add_net_unixgram instead.
```

Reply:
Ouch, fixed, thanks!